tests/framework/validators/DateValidatorTest.php

```diff
@@ -200,10 +195,6 @@ public function testValidateAttributeICUFormat($timezone)
             $model->attr_timestamp
         );
         $val = new DateValidator(['format' => 'yyyy-MM-dd']);
-        $model = FakedValidationModel::createWithAttributes(['attr_date' => []]);
-        $val->validateAttribute($model, 'attr_date');
-        $this->assertTrue($model->hasErrors('attr_date'));
-        $val = new DateValidator(['format' => 'yyyy-MM-dd']);
```

Comment:
These are tests for empty value parsing failure, which contradicts the issue #5108.
This fix can not be applied keeping them.